### PR TITLE
add new example with prisma smart subscriptions apollo

### PR DESCRIPTION
**Pull request by Cauen**
Once Apollo is also popular, i've added a new example based on [prisma-subscriptions example](https://github.com/hayes/pothos/tree/d0d2bdab01681d40d35e44549be582b40eda53aa/examples/prisma-subscriptions) with a working integration between:
* Pothos
* Smart Subscriptions Plugin
* Apollo v4

It will make it easier for other people to configure their projects and avoid mistakes like mine #748

### CODE REVIEW

File: examples/prisma-smart-subscriptions-apollo/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,41 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+generator client {
+  provider = "prisma-client-js"
```
Comment (hayes) [inline examples/prisma-smart-subscriptions-apollo/prisma/schema.prisma:7]: this looks to be missing the output location.  In a mono-repo we can't generate the client into the default location

Comment (Cauen) [reply examples/prisma-smart-subscriptions-apollo/prisma/schema.prisma:7]: Just fixed e.e

**Review by hayes — APPROVED** [review]
Looks good, thanks for adding this example! Should be very useful for anyone trying to build subscriptions with apollo!

**Comment by hayes** [comment]
Looks like there are a few eslint issues to fix, but the rest of the tests and type checking seem to be passing now!

**Comment by Cauen** [comment]
Forgot to install eslint on wsl xD
I think its now ok :D

**Pull request merged**